python/idsse_common/idsse/common/grid_proj.py

```diff
@@ -0,0 +1,113 @@
+"""Module that wraps pyproj (cartographic/geodetic) library and transforms objects into other data forms"""
+# -------------------------------------------------------------------------------
+# Created on Mon Jul 31 2023
+#
+# Copyright (c) 2023 Colorado State University. All rights reserved. (1)
+#
+# Contributors:
+#     Mackenzie Grimes (1)
+#
+# -------------------------------------------------------------------------------
+# pylint: disable=invalid-name
+
+from typing import Self, Tuple, Union, Any, Optional
+from enum import Enum, auto
+from math import floor
+
+from pyproj import CRS, Transformer
+from pyproj.enums import TransformDirection
+
+
+class PixelRounding(Enum):
```

Comment:
Nice to have: could we make this so the caller to _to_pixel could use strings, say 'ROUND', 'round', 'FLOOR', or 'floor'?